### PR TITLE
Handle pre-existing checkbox

### PR DESCRIPTION
**Pull request by Shreets**
->Show error message when no option is checked
-> Uncheck the rest option when none is checked and vice versa

### CODE REVIEW

File: src/pages/PreexistingCondition.js [hunk 1]
```diff
@@ -9,24 +9,84 @@ const PeexistingCondition = ({
...
+
+  const handleOnConditionClick = (item) => {
+    if (isConditionChecked) {
+      setIsConditionChecked(false);
+    }
+    if (isConditionChecked) {
+      setConditionError(false);
+    }
+
+    if (item === "noPrexistingCondition") {
```
Comment (nastikue) [inline src/pages/PreexistingCondition.js:46]: please move "noPrexistingCondition" to const

File: src/pages/PreexistingCondition.js [hunk 2]
```diff
@@ -9,24 +9,84 @@ const PeexistingCondition = ({
...
+        [item]: isConditionChecked,
+        noPrexistingCondition: false,
+      });
+    }
+  };
+
+  const validateForm = () => {
+    const isAnyTrue = Object.keys(preExistingCondition)
+      .map((key) => preExistingCondition[key])
+      .some((v) => v === true);
```
Comment (nastikue) [inline src/pages/PreexistingCondition.js:73]:
```suggestion
      .some((key) => preExistingCondition[key])
```

File: src/pages/PreexistingCondition.js [hunk 3]
```diff
@@ -9,24 +9,84 @@ const PeexistingCondition = ({
...
   const onNext = () => {
+    const isValid = validateForm();
+    if (!isValid) {
+      return;
+    }
+
     setPage(page + 1);
   };
 
+  console.log(intakeState);
```
Comment (nastikue) [inline src/pages/PreexistingCondition.js:89]: please remove console

File: src/pages/PreexistingCondition.js (hunk 1 above)
Comment (Shreets) [reply src/pages/PreexistingCondition.js]: this change has been made

File: src/pages/PreexistingCondition.js (hunk 2 above)
Comment (Shreets) [reply src/pages/PreexistingCondition.js]: this change has been made

File: src/pages/PreexistingCondition.js (hunk 3 above)
Comment (Shreets) [reply src/pages/PreexistingCondition.js]: console removed

File: src/pages/PreexistingCondition.js [hunk 4]
```diff
@@ -9,21 +10,80 @@ const PeexistingCondition = ({
...
+        ...intakeState,
+        [item]: isConditionChecked,
+        noPrexistingCondition: false,
+      });
+    }
+  };
+
+  const validateForm = () => {
+    const isAnyTrue = Object.keys(preExistingCondition)
+      .map((key) => preExistingCondition[key])
```
Comment (nastikue) [inline src/pages/PreexistingCondition.js:73]: .map((key) => preExistingCondition[key])
is redundant here

Comment (Shreets) [reply src/pages/PreexistingCondition.js]: map function removed

**Review by nastikue — APPROVED** [review]

**Pull request merged**